pages/portfolio.html

```diff
@@ -4,9 +4,81 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title class="title">My Portfolio</title>
+  <link rel="stylesheet" type="text/css" href="/stylesheets/styles.css" />
 </head>
 <body>
-  
+  <nav class="nav-block">
+    <ul class="main-nav">
+      <li>
+        <a href="index.html">Home</a>
+      </li>
+      <li>
+        <a href="about.html">About</a>
+      </li>
+      <li>
+        <a href="portfolio.html">My Portfolio</a>
+      </li>
+      <li class="right-side">
+        <a href="index.html">Kristin Lee</a>
+      </li>
+
+    </ul>
+  </nav>
+
+  <header>
+    <section class="header-cover" id="top">
+      <h2 class="cover-slogan">Kristin's Portfolio</h2>
+      <h3 class="cover-desc">Check out my projects below. Enjoy!</h3>
```

Comment:
It would make sense to use `h1` and `h2` here, and then use the classes to apply the desired styling